pr-review.ts

```diff
@@ -1 +1,36 @@
 
+
+interface User {
+  id: number;
+  name: string;
+  email?: string;
+}
+
+
+
+
+function getUserName(user: User: string {
```

Comment:
_:warning: ISSUE_

**Syntax Error**

There is a syntax error in the function declaration.

```diff
- function getUserName(user: User: string {
+ function getUserName(user: User): string {
```